app/models/privacy_policy.rb

```diff
@@ -0,0 +1,3 @@
+class PrivacyPolicy < ActiveRecord::Base  
+  self.table_name = 'privacy_policy'
```

Comment:
We probably want to require that a privacy policy at a minimum includes some text in both languages.

```suggestion
  validates :description, :spanish_description, presence: true

  self.table_name = 'privacy_policy'
```

Reply:
done